README.md

```diff
@@ -17,6 +17,17 @@ If you have a local copy of the ISO, you can point to it with the `--iso` parame
 
 You can also add additional kernel command line arguments (e.g. `"console=ttyS0"`) to the generated configuration files using the `--extra-args` parameter.
 
+## Usage
+
+Copy the files generated under the interim folder `/tmp/tmpxxx/ubuntu-installer/`
+to your tftp root folder for netboot, for example `/var/lib/tftpboot`:
+
+```
+$ sudo cp /tmp/tmpxxx/ubuntu-installer/* /var/lib/tftpboot/
+```
+
+Then your netboot server is ready to go.
+
```

Comment:
We should probably note that you still need to setup a DHCP server - though we can probably just point to an external document on that (IIRC, Ubuntu discourse describes this?)

Reply:
Thanks for the reminder. Yes, [the discourse page](https://discourse.ubuntu.com/t/netbooting-the-live-server-installer-via-uefi-pxe-on-arm-aarch64-arm64-and-x86-64-amd64/19240) describes this. This tool (and its README) will be updated in the discourse page, and it is a good idea to refer to the discourse page so they could mutually refer to each other.